### ISSUE TITLE
Fix documentation for apt-get-sqq parser

### ISSUE THREAD

**Issue by kellyjonbrazil** [kellyjonbrazil/jc#555]
The [documentation](https://kellyjonbrazil.github.io/jc/docs/parsers/apt_get_sqq) would be better with something like this as an usage example:

```bash
$ apt-get -sqq upgrade | jc --apt-get-sqq
```

or

```bash
$ jc apt-get -sqq full-upgrade
```

Missing out `upgrade` or `dist-upgrade` or `full-upgrade` from the command results in a command that fails:

```bash
apt-get -sqq 
E: Command line option 's' [from -sqq] is not understood in combination with the other options.
```

_Originally posted by @chriscroome in https://github.com/kellyjonbrazil/jc/issues/340#issuecomment-2026320141_

**Comment by chriscroome** [comment]
Thanks @kellyjonbrazil it is working well:

```bash
apt-get -sqq upgrade | jc --apt-get-sqq -Py
```

```yaml
---
- operation: unpack
  package: firefox-nightly-l10n-en-gb
  broken: 126.0a1~20240328092313
  proposed_pkg_ver: 126.0a1~20240328213634 namespaces/moz-fx-productdelivery-pr-38b5/repositories/mozilla:mozilla
  existing_pkg_ver:
  architecture: all
- operation: unpack
  package: firefox-nightly
  broken: 126.0a1~20240328092313
  proposed_pkg_ver: 126.0a1~20240328213634 namespaces/moz-fx-productdelivery-pr-38b5/repositories/mozilla:mozilla
  existing_pkg_ver:
  architecture: amd64
- operation: configure
  package: firefox-nightly-l10n-en-gb
  broken:
  proposed_pkg_ver: 126.0a1~20240328213634 namespaces/moz-fx-productdelivery-pr-38b5/repositories/mozilla:mozilla
  existing_pkg_ver:
  architecture: all
- operation: configure
  package: firefox-nightly
  broken:
  proposed_pkg_ver: 126.0a1~20240328213634 namespaces/moz-fx-productdelivery-pr-38b5/repositories/mozilla:mozilla
  existing_pkg_ver:
  architecture: amd64
```

**Comment by kellyjonbrazil** [comment]
Updates in the `dev` branch and will make it in the next release.

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/apt_get_sqq.py

**Comment by kellyjonbrazil** [comment]
Fixed in v1.25.3
